uefi/src/proto/tcg/v1.rs

```diff
@@ -214,6 +214,19 @@ impl Debug for PcrEvent {
     }
 }
 
+// Manual `PartialEq` implementation since it can't be derived for a packed DST.
+impl PartialEq for PcrEvent {
+    fn eq(&self, rhs: &Self) -> bool {
+        self.pcr_index() == rhs.pcr_index()
+            && self.event_type() == rhs.event_type()
+            && self.digest == rhs.digest
+            && self.event_data_size == rhs.event_data_size
+            && self.event_data == rhs.event_data
+    }
+}
+
+impl Eq for PcrEvent {}
```

Comment:
nit: Not sure but I think Eq can be derived automatically, even with a custom PartialEq impl 🤔

Reply:
You're right, updated.